src/diffusers/utils/loading_utils.py

```diff
@@ -32,6 +32,8 @@ def load_image(
             raise ValueError(
                 f"Incorrect path or URL. URLs must start with `http://` or `https://`, and {image} is not a valid path."
             )
+    elif isinstance(image, PIL.Image.Image):
+        image = image
```

Comment:
Wonder why would someone call `load_image()` on an already properly loaded image, though.

Reply:
Many tutorials are written like this😁, Perhaps to ensure things like RGB